src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyNameValidatorTest.java

```diff
@@ -63,7 +63,7 @@ public void inputStudyNameWithDigitsAndWhiteSpacesIsValid() throws JobParameters
     }
 
     @Test
-    public void inputStudyNameWithSymbolsIsValid() throws JobParametersInvalidException {
+    public void inputStudyNameWithSymbolsIsNotValid() throws JobParametersInvalidException {
```

Comment:
This should be renamed to `inputStudyNameWithSymbolsIsValid` because the exception is no longer expected.